invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/components/ResultsList/ResultsList.js

```diff
@@ -92,12 +93,16 @@ export class ResultsList extends Component {
   prepareData(data) {
     return data.hits.map(row => {
       let serialised = formatter.item.toTable(row);
-      serialised['Actions'] = this.actions(row, row.status);
-      delete serialised['Created'];
-      delete serialised['Updated'];
-      delete serialised['Internal location'];
-      delete serialised['Document ID'];
-      return serialised;
+      serialised['Actions'] = this.actions(row, row.metadata.status);
```

Comment:
Here is correctly used with `let` :)

Reply:
Not to be too pedantic, but I see we use both serialised and serialized. Perhaps we should rename one of them so it's easier to find all uses if we are searching for it?